webapp/src/components/common/SplitView/index.tsx

```diff
@@ -55,16 +55,16 @@ function isValidSizes(sizes: unknown): sizes is number[] {
  * @returns A React component displaying a split layout view with resizable panes.
  */
 function SplitView({
-  id,
+  splitId,
   children,
   direction = "horizontal",
-  sizes,
+  sizes = [10, 90], // Most common split ratio: small sidebar (10%) + main content area (90%)
```

Comment:
It's small for the left panel of certain views. You need to test with a 15" size.

Reply:
I tested it and it's more than enough on laptop screens. It's resizable so users can adjust as needed.